### PR TITLE
20 09lesson

### PR DESCRIPTION
**Pull request by nikitakalenkevich**
Снимок экрана 2023-09-24 214754
Снимок экрана 2023-09-24 214645

### CODE REVIEW

File: homework/basic_html_layout/ttx.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+<body>
+    <style>
+        .box {
+            display: inline-block;
+            width: 800px;
+            height: 85px;
+            position: relative;
+            background-color: grey;
+            position: relative;
```
Comment (kalenkevich) [inline homework/basic_html_layout/ttx.html]: You can remove `position: relative;` as it is already defined above.

**Review by kalenkevich — COMMENTED** [review]
Please check my comment below.

File: homework/basic_html_layout/ttx.html (hunk 1 above)
Comment (nikitakalenkevich) [reply homework/basic_html_layout/ttx.html]: thanks , fixed.

**Review by kalenkevich — APPROVED** [review]
Nice work!!!

**Pull request merged**